onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -464,7 +461,7 @@ including arg name, arg type (contains both type and shape).)pbdoc")
               for (int i = 0; i < shape->dim_size(); ++i) {
                 if (shape->dim(i).has_dim_value()) {
                   res << shape->dim(i).dim_value();
-                } else if (shape->dim(i).has_dim_param()) {
+                } else {
                   res << "None";
```

Comment:
>"None"; [](start = 25, length = 7)

should this output dim_param instead of None? It would be confusing when model like BiDAF who contains multiple symbolic dims.

Reply:
That seems reasonable. I don't know the history as to why 'None' was chosen but using dim_param if set would make more sense to me.